server.js

```diff
@@ -5769,6 +5769,49 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Jetbrains Plugins repository integration
+camp.route(/^\/jetbrainsplugins\/([^\/]+)\/(d)\.(svg|png|gif|jpg|json)$/,
```

Comment:
Could I ask you to make the URI consistent with `https://img.shields.io/wordpress/plugin/v/akismet.svg`?

i.e. `/jetbrains/plugin/d/name.of.plugin.svg`